packages/vaadin-notification/test/renderer.test.js

```diff
@@ -47,16 +47,6 @@ describe('renderer', () => {
       };
     });
 
-    it('should remove template when added after renderer', () => {
```

Comment:
It is a no longer relevant test since we made the template renderer responsible for throwing such exceptions.

I missed this test while removing the Template API as far as it successfully passes by the reason of another exception (`notification._observer.flush()` throws an exception since we don't have `_observer` defined).